lgn-provers/src/params/mod.rs

```diff
@@ -319,13 +237,9 @@ impl ParamsLoader
                 expected_hashes_file.to_path_buf(),
             ),
         );
-        debug!(
-            "expected hashes from: {:?} is {:?}",
-            expected_hashes_file, expected_hashes
-        );
```

Comment:
I think this were left over from debugging. This is making the logs quite verbose, so I merged them into a single log entry.